test/simulator-e2e-specs.js

```diff
@@ -260,6 +261,36 @@ function runTests (deviceType) {
       (await sim.hasCalendarAccess(BUNDLE_ID)).should.be.false;
     });
 
+    it('should properly start simulator in headless mode on Xcode9+', async function () {
+      const sim = await getSimulator(udid);
```

Comment:
These should be revered. There is no reason to have the sim object if we are going to skip.